src/basic/string-util.c

```diff
@@ -1043,34 +1043,29 @@ char* strrep(const char *s, unsigned n) {
         return r;
 }
 
-int split_pair(const char *s, const char *sep, char **l, char **r) {
-        char *x, *a, *b;
-
+int split_pair(const char *s, const char *sep, char **ret_first, char **ret_second) {
```

Comment:
Hmm, I reckon this can be fully replaced by extract-words?

Reply:
The functions in env-utils.c also uses `split_pair()`, and use neither `extract_first_word()` nor `extract_many_words()`. The function `split_pair()` is simple enough and suitable for handling environment variable assignment.